lib/ts/types.ts

```diff
@@ -113,3 +114,13 @@ export type User = {
     // the recipeUserId can be converted to string from the RecipeUserId object type.
     toJson: () => any;
 };
+
+export type SSRSessionContextType = {
+    isContextFromSSR: true;
+    loading: false;
+    doesSessionExist: boolean;
+    accessTokenPayload: any;
+    invalidClaims: ClaimValidationError[]; // TODO: Find if it's possible to compute invalidClaims on BE
```

Comment:
As mentioned in the auth-react PR, if we can't find a way to do these properly (which I don't think we can, not right now at least), the claim validation related props should be removed from this type.